tests/test_config.py

```diff
@@ -614,6 +630,33 @@ def test_client_options(self):
         conn = connector.Connector.from_config(config_obj)
         self.assertEqual(50, conn.doc_managers[0].mongo.max_pool_size)
 
+    def test_ssl_options(self):
+        config_def = {
+            'mainAddress': 'localhost:27017',
+            'oplogFile': from_here('lib', 'dummy.timestamp'),
+            'ssl': {
+                'sslCertfile': 'certfile.pem',
+                'sslKeyfile': 'certfile.key',
+                'sslCACerts': 'ca.pem'
+            },
+        }
+        for cert_policy, expected_ssl_cert_req in [
+                ('ignored',  ssl.CERT_NONE),
+                ('optional', ssl.CERT_OPTIONAL),
```

Comment:
The current docs say ssl_cert_reqs "must be one of the three values ssl.CERT_NONE (certificates ignored), ssl.CERT_REQUIRED (certificates required and validated), or ssl.CERT_OPTIONAL (the same as CERT_REQUIRED, unless the server was configured to use anonymous ciphers)."

That sounds like there's still a use-case for CERT_OPTIONAL.